admin-site/src/EventsList.svelte

```diff
@@ -12,6 +12,7 @@
   import { pageStateInStore, focusedEventStore, indexInEventsStore, focusedEventLabelStore} from './PageStore.js';
 
   const dispatch = createEventDispatcher();
+  const dispatchState = createEventDispatcher();
```

Comment:
Is there a reason you've created a second event dispatcher rather than using the extant one?

Reply:
I thought I'd avoid updating/changing states and variables unnecessarily. I'll put it all in one though, and see if anything weird happens.

Reply:
I generally agree in terms of 'be careful about making changes to existing functionality'. In this case my impression from Svelte is that the dispatch is pretty generic, all the specificity is in the messages...